orchestration/scripts/request-release-date-snapshot.py

```diff
@@ -44,5 +44,7 @@ def submit_snapshot(**kwargs):
   }
 }]
 
-job_id = submit_snapshot(name=snapshot_name, description=snapshot_description, contents=snapshot_contents, profileId=profile_id, readers=[reader_email])
+reader_emails = ["clingendevs@firecloud.org"] if environment == "prod" else []
```

Comment:
was it intentional to make reader email hardcoded rather than drawing it from the `READER_EMAIL` env var as part of this change?

Reply:
It was hardcoded a level up in the argo template 🤷🏾 making this change because the dev environment doesn't really need any readers, and I fear trying to pass a "none" type object from argo to Python